app/views/previewers/create.js.erb

```diff
@@ -12,4 +12,5 @@ http://digitalnz.org/supplejack
 %>
 
 $("#preview-area").html('<%= j render "previewer" %>');
-PreviewJobsPoller.poll();
\ No newline at end of file
+$(document).foundation();
```

Comment:
`foundation() ` load happen hear. I saw that in your application.js. Maybe it is ok.

Reply:
Yeah I need this because this document gets ajaxed in and foundation is not yet bound.